### PR TITLE
Update config [ECR-4151]

### PR DESCRIPTION
**Pull request by slowli**
This PR updates the *Configuration* article.

See: https://jira.bf.local/browse/ECR-4151

### CODE REVIEW

File: src/architecture/configuration.md [hunk 1]
```diff
@@ -6,118 +6,109 @@
...
+    separated into an independent executable).
+
+Global configuration may be changed using [the supervisor service](../advanced/supervisor.md),
+and local configuration [by editing the configuration file](#changing-configuration).
+When the blockchain is being created, the initial global configuration
+is supplied from a file, too, since the blockchain is absent at this point.
+
+## Genesis Configuration
+
+**Genesis configuration** is the global configuration necessary to start
```
Comment (aleksuss) [inline src/architecture/configuration.md]:
```suggestion
**Genesis configuration** is the global configuration that necessary to start
```

File: src/architecture/configuration.md [hunk 2]
```diff
@@ -6,118 +6,109 @@
...
 
 This categorization holds for both core and service parameters.
+The following table shows three possible parameter categories.
+
+| Scope   | Core | Anchoring service |
+|---------|------|-------------------|
+| Global  | Validators’ public keys | Anchoring address |
+| Local   | Validator’s private key | (N/A) |
+
+Note that there is no unified mechanism for local service configuration.
```
Comment (aleksuss) [inline src/architecture/configuration.md]:
```suggestion
Note that there is no unified mechanism for a changing local service configuration.
```
?

File: src/architecture/configuration.md [hunk 3]
```diff
@@ -6,118 +6,109 @@
...
+and local configuration [by editing the configuration file](#changing-configuration).
+When the blockchain is being created, the initial global configuration
+is supplied from a file, too, since the blockchain is absent at this point.
+
+## Genesis Configuration
+
+**Genesis configuration** is the global configuration necessary to start
+a new node. While nodes update the global configuration using
+the supervisor service, a node need to get the genesis configuration from
+a trusted source. Indeed, at the point the genesis config is used, the node
```
Comment (aleksuss) [inline src/architecture/configuration.md]:
```suggestion
a trusted source. Indeed, at this point the genesis config is used, the node
```

File: src/architecture/configuration.md (hunk 1 above)
Comment (popzxc) [reply src/architecture/configuration.md]: I think it's okay without "that" here. Or, if not, it should be "that is".

File: src/architecture/configuration.md (hunk 3 above)
Comment (popzxc) [reply src/architecture/configuration.md]: This is not correct, IMHO. We are talking about **the** point *when* config will be used.

**Pull request merged**